### PR TITLE
fix egress route for ipv6

### PR DESCRIPTION
**Pull request by yabinma**
## Checklist before requesting a review
- [x] My changes affect only 10 files or less.
- [x] I have performed a self-review of my code and tested it.
- [ ] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [x] If it is a bugfix, my code is <= 200 lines.
- [x] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [ ] My unit tests pass locally.
- [x] Netclient & Netmaker are awesome.

### CODE REVIEW

File: firewall/firewall_linux.go [hunk 1]
```diff
@@ -58,7 +59,17 @@ func getInterfaceName(dst net.IPNet) (string, error) {
 	}
 	routes, err := h.RouteGet(dst.IP)
 	if err != nil {
-		return "", err
+		if !strings.Contains(err.Error(), "network is unreachable") {
+			return "", err
+		}
+		//if ipv4 address is unreachable, try ipv6 address
+		dst.IP = net.ParseIP("2606:4700:4700::1111")
```
Comment (abhishek9686) [inline firewall/firewall_linux.go:66]: why is this address hard coded?

File: functions/localport_linux.go [hunk 2]
```diff
@@ -12,7 +13,16 @@ func getDefaultInterface() (string, error) {
 	// routes[0] will be default route
 	routes, err := netlink.RouteGet(dest)
 	if err != nil {
-		return "", err
+		if !strings.Contains(err.Error(), "network is unreachable") {
+			return "", err
+		}
+		//if ipv4 address is unreachable, try ipv6 address
+		dest = net.ParseIP("2606:4700:4700::1111")
+		// routes[0] will be default route
```
Comment (abhishek9686) [inline functions/localport_linux.go:21]: hard coded addr

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: firewall/firewall_linux.go (hunk 1 above)
Comment (yabinma) [reply firewall/firewall_linux.go]: Good point @abhishek9686 .  2606:4700:4700::1111 is the ipv6 format for 1.1.1.1.  
I added a commit and changed it to constant.  Please have a review again. Thanks.

**Comment by yabinma** [comment]
Close this one and open a new PR for avoiding two PRs.

**Pull request closed without merging**